CougSat1-PMIC/src/PMICConfiguration.h

```diff
@@ -149,4 +149,48 @@
 #define THERM_CAL_PV3_FRONT_BOTTOM_270K (0.158)
 #define THERM_CAL_PV3_FRONT_BOTTOM_350K (0.852)
 
+
+// Comparison Constant for Thermistor readings in celcius
+#define TEMP_MIN (0)
+#define TEMP_MAX (50)
+#define REG_TEMP_MAX (85)
+
+// Priority Levels
+#define priority_PR3V3_one      100
```

Comment:
Terrible naming conventions, I can't tell where these are being used. Read the code SOP